client/src/assets/stylesheets/partials/_lesson.scss

```diff
@@ -10,6 +10,12 @@ $letterSize: 8em;
   padding: 2em 0;
 }
 
+.lesson-progress {
+  height: 5px;
+  border-radius: 10px;
+  background-color: #2bc252;
```

Comment:
We use SASS for a reason :) No hardcoding hex colors.  

Create a configurable variable instead.  See _base.scss where we declare `$baseBg`.  Something like `$progress-color` would be solid here. 

Let's start good habits early!